deps.edn

```diff
@@ -62,7 +62,8 @@
   {:extra-paths ["test" "dev-resources"]
    :extra-deps  {lambdaisland/kaocha         {:mvn/version "1.71.1119"}
                  org.clojure/test.check      {:mvn/version "1.1.1"}
-                 com.magnars/test-with-files {:mvn/version "2021-02-17"}}
+                 io.github.cap10morgan/test-with-files {:git/tag "v1.0.0"
+                                                        :git/sha "9181a2e"}}
```

Comment:
I assume a bug? Might be worth a comment, so in the future we know when/if we can revert to the main repo.

Reply:
Nope. The explanation is in the commit message. I forked it to add a handy debugging feature (namely, keeping the tmp dir around after this goes out of scope). Given the age of at least one other PR upstream, I doubt it's worth the trouble to try to get it accepted upstream. Between that and the fact that it's strictly a dev / test dep, I wasn't going to worry about it too much.

Happy to move the fork to the Fluree GH org though, if anyone would prefer that. It probably does make more sense in this case.